l10n_ar_afipws/wizard/res_partner_update_from_padron_wizard_view.xml

```diff
@@ -12,43 +12,43 @@
         <field name="model">res.partner.update.from.padron.wizard</field>
         <field name="arch" type="xml">
             <form string="Update From Padron AFIP">
-                <field invisible="1" name="state"/>
+                <field column_invisible="1" name="state"/>
                 <header/>
                 <sheet>
-                    <group attrs="{'invisible': [('state', '!=', 'finished')]}" col="1">
+                    <group invisible="state != 'finished'" col="1">
                         <h2>There are no more partners to update for this request...</h2>
                     </group>
-                    <p attrs="{'invisible': [('state', '!=', ('option'))]}" class="oe_grey">Only Partners with cuit are going to be updated.
+                    <p invisible="state != 'option'" class="oe_grey">Only Partners with cuit are going to be updated.
                         <br/>
                         Select the list of fields you want to update.</p>
-                    <group attrs="{'invisible': [('state', 'not in', ('option',))]}">
+                    <group invisible="state != 'option'">
                         <field name="field_to_update_ids" options="{'no_create': True}" widget="many2many_tags"/>
                         <field name="update_constancia"/>
                         <field name="title_case"/>
                     </group>
-                    <group attrs="{'invisible': [('state', 'in', ('option', 'finished'))]}" col="1">
+                    <group invisible="state != 'option'" col="1">
```

Comment:
La condicion era multiple. ¿Era necesario cambiarlo?
Sino puede quedar como era
`<group invisible="state in ('option', 'finished')" col="1">`